src/Experience/experience.js

```diff
@@ -0,0 +1,128 @@
+        let techIntro = <StanTechTop />
+        let portfolio = <Portfolio />
+        let rent = <ArtStuff />
+        return(
+            <div className='container'>
+                <div className='card-wrapper'>
+                    <ExperienceCard
+                        logo={require('../assets/logo-white.png')}
+                        logoAlt='Artist Connect'
+                        image={require('../assets/startup-weekend.png')}
+                        title='1st Place Startup Weekend'
+                        content={arts}
+                    />
+                    <ExperienceCard 
+                        logo={require('../assets/stantech-logo.png')}
+                        logoAlt='StanTech'
+                        title='Complete Site Redesign'
+                        topContent={techIntro}
+                        content={tech}
+                    />
+                    <ExperienceCard 
+                        logo={require('../assets/ck-logo.png')}
+                        logoAlt='Caren Keyes original portfolio'
+                        title='Original Portfolio Site'
+                        image={require('../assets/portfolio.png')}
+                        logoAlt='original portfolio website'
+                        content={portfolio}
+                    />
+                    <ExperienceCard 
+                        logo={require('../assets/rentartstufflogo2.png')}
+                        logoAlt='Rent Art Stuff'
+                        title='Rent Art Stuff MVP'
+                        image={require('../assets/artstuff.png')}
+                        content={rent}
+                    />
+
+                </div>
+            </div>
+        )
+    }
+}
+
+const ArtsConnect = () => {
+    return(
+        <div className='arts-connect expanded-content'>
+            <div>
+                <p>
+                I had the opportunity to be the only developer on an amazing team or artists and entrepreneurs.                </p>
+                <p>
+                Over about two days, I built out a demo of our space and gear sharing platform for all kinds of artists.                 </p>
+                <div className='card-links'>
+                    <p><a href='https://www.reviewjournal.com/business/entrepreneurs/artist-connect-takes-first-place-at-startup-weekend-las-vegas-2018-1531226/' target='_blank'>NEWS ARTICLE</a></p>
+                    <p><a href='http://www.artistconnect.co/' target='_blank'>PROTOTYPE</a></p>
+                </div>
+            </div>
+        </div>
+    )
+}
+
+const StanTechTop = () => {
+    return(
+        <div className='stantech-top'>
+            <div className='plant-animation'></div>
+        </div>
+    )
+}
+
+const StanTech = () => {
+    return(
+        <div className='stan-tech expanded-content'>
+            <div>
+                <p>I’ve been asked to do a complete redesign of the StanTech website. The commission is to “make it pop.”
```

Comment:
Not a code review thing, but "make it pop" as a requirement is scary! So much room for failure because someone's opinion is different than yours. Be super careful with this.... Getting early feedback and iterating will help make sure you aren't surprised later.